### ISSUE TITLE
Make front-page.php not a selectable template

### ISSUE THREAD

**Issue by rcass** [redacademy/fox-summer-2017#4]
By adding this code block:
![screen shot 2017-09-14 at 11 09 41 am](https://user-images.githubusercontent.com/5028555/30446190-3d8bf392-993d-11e7-86e8-9ad734f6dbf2.png)

It makes the front-page template as a selectable page template. 

Since we want the front-page.php to always be the template used for the front page, we should not reply on the user to selecting it (or deselecting it by mistake).

Without that code block front-page.php will be the template used for the front page (because of the template hiearchy).

Basically, please remove the `Template Name: Front-page` so it no longer becomes a UI selectable template.

**Comment by DHilscher** [comment]
I removed it. Thanks Rose!
